src/components/common/back/Back.jsx

```diff
@@ -1,18 +1,20 @@
-import React from "react"
-import { useLocation } from "react-router-dom"
+import React from 'react';
+import { useLocation } from 'react-router-dom';
 
 const Back = ({ title }) => {
-  const location = useLocation()
+	const location = useLocation();
 
-  return (
-    <>
-      <section className='back'>
-        <h2>Home / {location.pathname.split("/")[1]}</h2>
-        <h1>{title}</h1>
-      </section>
-      <div className='margin'></div>
-    </>
-  )
-}
+	return (
+		<>
+			<section className="back">
+				<div className="mycontainer">
+					<h2>Home / {location.pathname.split('/')[1]}</h2>
+					<h1>{title}</h1>
+				</div>
+			</section>
+			{/* <div className='margin'></div> */}
+		</>
+	);
+};
```

Comment:
I don't want this back section in all pages, instead you should be doing this at every page.
![image](https://github.com/LernTek/LernTek-Main/assets/74200389/b8444b3e-7aa0-4df6-867d-8809345157dc)